components/cdc/src/initializer.rs

```diff
@@ -274,6 +274,7 @@ impl<E: KvEngine> Initializer<E> {
             assert!(!has_remain);
             let mut locks = BTreeMap::<Key, MiniLock>::new();
             for (key, lock) in key_locks {
+                // todo: shall we also consider `LockType::Lock` here ?
```

Comment:
Seems it's also ignored in 7bf8ca33d. It's better to comment why `Lock` can be ignored.